test_scripts/API/VehicleData/HandsOffSteering/034_SubscribeVD_during_resumption_IGN_OFF_ON_for_two_Apps.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0257-New-vehicle-data-HandsOffSteering.md
+--
+-- Description: Check that SDL restored SubscribeVehicleData on 'handsOffSteering' parameter after IGN_OFF/IGN_ON cycle
+-- for two Apps
+--
+-- Preconditions:
+-- 1) Update preloaded_pt file, add handsOffSteering parameter to VD_RPC group
+-- 2) RPC SubscribeVehicleData and handsOffSteering is allowed by policies
+-- 3) App_1 and App_2 are registered and subscribed on handsOffSteering VD
+-- 4) IGN_OFF/IGN_ON cycle is performed
+-- 5) App_1 re-registers with actual HashId
+-- SDL does:
+-- - a) send VehicleInfo.SubscribeVehicleData request to HMI during resumption
+-- - b) process successful response from HMI
+-- - c) respond RAI(SUCCESS) to mobile app
+-- 6) App_2 re-registers with actual HashId
+-- SDL does:
+-- - a) not send VehicleInfo.SubscribeVehicleData request to HMI during resumption
+-- - b) respond RAI(SUCCESS) to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/HandsOffSteering/common')
+
+--[[ Local Variables ]]
+local rpc_sub = "SubscribeVehicleData"
+local onVDValue = true
+local appId_1 = 1
+local appId_2 = 2
+local isExpectedSubscribeVDonHMI = true
+local notExpectedSubscribeVDonHMI = false
+local isHMISubscription = true
```

Comment:
Please use isExpectedSubscribeVDonHMI and notExpectedSubscribeVDonHMI insted of isHMISubscription and notHMISubscription